network_modeling/configs/templates/1.5/aruba/common/acl.j2

```diff
@@ -5,6 +5,7 @@ access-list ip mgmt
 {% set sequence = sequence+10 %}    {{ sequence }} permit tcp {{ variables.HMN_NETWORK_IP }}/{{ variables.HMN_NETMASK }} any eq https
 {% set sequence = sequence+10 %}    {{ sequence }} permit udp {{ variables.HMN_NETWORK_IP }}/{{ variables.HMN_NETMASK }} any eq snmp
 {% set sequence = sequence+10 %}    {{ sequence }} permit udp {{ variables.HMN_NETWORK_IP }}/{{ variables.HMN_NETMASK }} any eq snmp-trap
+{% set sequence = sequence+10 %}    {{ sequence }} permit udp any eq ntp {{ variables.CMN_NETWORK_IP }}/{{ variables.CMN_NETMASK }}
```

Comment:
This should be `NMN_NETWORK_IP` and `NMN_NETMASK`.

Looking at the generated configuration the servers are added using the NMN IPs in the CSM VRF (which is correct)
```
ntp server 10.252.1.9
ntp server 10.252.1.8
ntp server 10.252.1.7
ntp enable
ntp vrf CSM
```
However the ACL permits access from the CMN not the NMN.
```
access-list ip mgmt
...
    60 permit udp any eq ntp 10.102.193.0/255.255.255.128
```